packages/lib/src/vis/raw/RawVis.tsx

```diff
@@ -1,13 +1,23 @@
 import styles from './RawVis.module.css';
+import { isImage } from './utils';
 
 const LARGE_THRESHOLD = 1_000_000;
 
 interface Props {
   value: unknown;
+  title?: string;
 }
 
 function RawVis(props: Props) {
-  const { value } = props;
+  const { value, title } = props;
+
+  if (value instanceof Uint8Array && isImage(value)) {
+    return (
+      <div className={styles.root}>
+        <img src={URL.createObjectURL(new Blob([value]))} alt={title} />
+      </div>
+    );
+  }
```

Comment:
From an architecture point of view, it would perhaps make more sense to add a specific `Vis` with `supportsDataset` equal to `value instanceof Uint8Array && isImage(value)` ?

Reply:
Sorry for not reporting on this in the issue, but after investigating, I realised that values are fetched by the visualization containers, so `supportsDataset` can't have access to it. And if I add an `Opaque` visualization for opaque datasets, then it would also have to handle datasets that don't contain JPEG/PNG, which would duplicate the `Raw` visualization. I could make an `OpaqueContainer` that renders a new `RawImageVis` if an image is detected, or `RawVis` if not, but it felt unnecessary.

Reply:
Ah yes, didn't realize the condition was on the value. Fine then.